src/document/operation/changeoperation.js

```diff
@@ -77,45 +80,15 @@ CKEDITOR.define( [ 'document/operation/operation', 'ckeditorerror' ], ( Operatio
 			}
 
 			// Remove or change.
-			if ( oldAttr !== null ) {
+			if ( oldAttr !== null && newAttr === null ) {
 				for ( value of this.range ) {
-					if ( !value.node.hasAttr( oldAttr ) ) {
-						/**
-						 * The attribute which should be removed does not exists for the given node.
-						 *
-						 * @error operation-change-no-attr-to-remove
-						 * @param {document.Node} node
-						 * @param {document.Attribute} attr
-						 */
-						throw new CKEditorError(
-							'operation-change-no-attr-to-remove: The attribute which should be removed does not exists for given node.',
```

Comment:
You removed both checks and errors (`operation-change-no-attr-to-remove` and `operation-change-attr-exists`), as we talked F2F you said they were problematic during the OT. The problem is that they are here because of reason: if the operation with the wrong `oldAttr` will be applied, then reversing that operation will now work properly. So the issue is not solved, it is hidden, and may be very hard to solve later. Also this check helped me in finding issues when I was working on the change delta multiple times, so I prefer it stays here.